docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,487 @@
+# Getting Started
```

Comment:
This isn't exactly a tutorial, it's more of a how-to / usage guide. Though I'd be interested to get Anne's take here.

Reply:
I agree, some part are more of a tutorial, like installing it and getting started with some basic layer configurations, while other tasks are like how-tos.

Shall we merge it first then split it up, or do the splitting-up in this PR? @benhoyt

Reply:
I'm in favour of incremental improvements, that is, merging this first pass, then improving in subsequent PRs. But let's see what Anne says.

Reply:
Hey @benhoyt , @IronCore864 , 

I agree that the updates can be incremental. But please give me some time to go through the changes in this PR before merging, for now.

Thanks!

Reply:
Yep, no problem -- take your time!